CoroutineTemplate/app/build.gradle.kts

```diff
@@ -0,0 +1,116 @@
+plugins {
+    id("com.android.application")
+
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("kotlin-parcelize")
+
+    id("dagger.hilt.android.plugin")
+    id("androidx.navigation.safeargs.kotlin")
+}
+
+val keystoreProperties = rootDir.loadGradleProperties("signing.properties")
+
+android {
+    signingConfigs {
+        create("config") {
+            // Remember to edit signing.properties to have the correct info for release build.
+            storeFile = file("../keystore_production")
+            storePassword = keystoreProperties.getProperty("KEYSTORE_PASSWORD") as String
+            keyPassword = keystoreProperties.getProperty("KEY_PASSWORD") as String
+            keyAlias = keystoreProperties.getProperty("KEY_ALIAS") as String
+        }
+    }
+
+    compileSdk = Versions.ANDROID_COMPILE_SDK_VERSION
+    defaultConfig {
+        applicationId = "co.nimblehq.coroutine"
+        minSdk = Versions.ANDROID_MIN_SDK_VERSION
+        targetSdk = Versions.ANDROID_TARGET_SDK_VERSION
+        versionCode = Versions.ANDROID_VERSION_CODE
+        versionName = Versions.ANDROID_VERSION_NAME
+    }
+
+    buildTypes {
+        getByName(BuildType.RELEASE) {
+            isMinifyEnabled = true
+            isDebuggable = false
+            isShrinkResources = true
+            proguardFiles(
```

Comment:
(Optional) Make this statement one line 🙏🏼
`proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")`

Reply:
too long I think :)

Reply:
@luongvo I think it's good to make it single line 🤔